x-pack/plugins/security_solution/public/cases/containers/use_delete_cases.tsx

```diff
@@ -90,10 +90,8 @@ export const useDeleteCases = (): UseDeleteCase => {
 
       const caseIds = cases.map((theCase) => theCase.id);
       // We don't allow user batch delete sub cases on UI at the moment.
-      if (cases[0].type != null || cases.length > 1) {
+      if (caseIds.length > 0) {
```

Comment:
@cnasikas do you agree with this new logic? We were confused with the old ones and mostly `cases.length > 1`

Reply:
I was confused also with the logic to be honest. Yes, much better!